dbt_subprojects/hourly_spellbook/models/_project/ens/ens_set_name_latest.sql

```diff
@@ -3,13 +3,15 @@
         schema = 'ens'
         ,alias = 'set_name_latest'
         
-        ,materialized = 'table'
+        ,materialized = 'incremental'
+        ,incremental_strategy = 'merge'
         ,file_format = 'delta'
         ,unique_key = ['address', 'name']
```

Comment:
Do you think the unique_key can be just `name`?

Reply:
likely, but the name + address combo will always be unique too. let's leave as-is.